airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -871,6 +871,149 @@ def record_process_components(self, record: MutableMapping[str, Any]) -> Optiona
                     yield customer_address
 
 
+class CustomerJourney(ShopifyBulkQuery):
+    """
+    Output example to BULK query `customer_journey_summary` from `orders` with `filter query` by `updated_at` sorted `ASC`:
+        {
+            orders(query: "updated_at:>='2020-01-20T00:00:00+00:00' AND updated_at:<'2024-04-25T00:00:00+00:00'", sortKey:UPDATED_AT) {
+                edges {
+                    node {
+                        __typename
+                        order_id: id
+                        createdAt
+                        updatedAt
+                        customerJourneySummary {
+                            ready
+                            momentsCount {
+                                count
+                                precision
+                            }
+                            customerOrderIndex
+                            daysToConversion
+                            firstVisit {
+                                id
+                                landingPage
+                                landingPageHtml
+                                occurredAt
+                                referralCode
+                                referrerUrl
+                                source
+                                sourceType
+                                sourceDescription
+                                utmParameters {
+                                    campaign
+                                    content
+                                    medium
+                                    source
+                                    term
+                                }
+                            }
+                            lastVisit {
+                                id
+                                landingPage
+                                landingPageHtml
+                                occurredAt
+                                referralCode
+                                referrerUrl
+                                source
+                                sourceType
+                                sourceDescription
+                                utmParameters {
+                                    campaign
+                                    content
+                                    medium
+                                    source
+                                    term
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    """
+
+    query_name = "orders"
+    sort_key = "UPDATED_AT"
+
+    visit_fields: List[Field] = [
```

Comment:
Is there are reason why we don't have [marketing_event](https://shopify.dev/docs/api/admin-graphql/2024-04/objects/CustomerVisit#field-customervisit-marketingevent)? Should we document that here?

Reply:
Yes, there is a reason here:
- https://github.com/airbytehq/product-request-backlog/issues/187#issue-2350915819

Reply:
Just to confirm: the reason is that it wasn't ask by the customer requesting this feature, right?

Reply:
Currently, yes. I'd love to add this, later on when there is a need for it.

Reply:
Cool! That makes a lot of sense to me. I just wanted to make sure that we had a trace of this. Thanks Baz!